src/devices/Pn532/ListPassive/TargetBaudRate.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Pn532.ListPassive
+{
+    /// <summary>
+    /// The baud rate target for specific cards
+    /// </summary>
+    public enum TargetBaudRate
+    {
+        /// <summary>
+        ///  106 kbps type A (ISO/IEC14443 Type A)
+        /// </summary>
+        B106kbpsTypeA = 0x00,
```

Comment:
Should those values be implied by PollingType?

Reply:
Those ones are specific to the ListPassiveTarget function. They are unfortunately not the same as for the AutoPoll one. I would have love to have only 1 type of baud rates.